x-pack/test/cloud_security_posture_functional/pages/cis_integrations/cspm/cis_integration_aws.ts

```diff
@@ -38,8 +38,7 @@ export default function (providerContext: FtrProviderContext) {
       await cisIntegration.navigateToAddIntegrationCspmPage();
     });
 
-    // FLAKY: https://github.com/elastic/kibana/issues/186438
-    describe.skip('CIS_AWS Organization Cloud Formation', () => {
+    describe('CIS_AWS Organization Cloud Formation', () => {
       it('Initial form state, AWS Org account, and CloudFormation should be selected by default', async () => {
         expect((await cisIntegration.isRadioButtonChecked('cloudbeat/cis_aws')) === true);
         expect((await cisIntegration.isRadioButtonChecked('organization-account')) === true);
```

Comment:
Should this even be FTR or could this be a unit test?

Reply:
hmmm, well this was added months ago
but now that u mentioned it, I think its a good idea to remove this from FTR